frontend/src/api/Game.ts

```diff
@@ -8,8 +8,6 @@ import { Color } from './Color';
 
 export type Player = {
   user: User,
-  code: string,
-  language: string,
```

Comment:
nit: change solved to be `boolean[]`

Reply:
Wait, why should allSolved be a boolean array? The backend still uses a Boolean to represent if all players have solved all problems.